infoblox_client/objects.py

```diff
@@ -337,6 +337,9 @@ def create_check_exists(cls, connector, check_if_exists=True,
     @classmethod
     def create(cls, connector, check_if_exists=True,
                update_if_exists=False, **kwargs):
+        if str(cls) == "<class 'infoblox_client.objects.DNSZone'>":
+            if 'fqdn'.lower() not in kwargs:
+                raise ib_ex.InfobloxMissingField(response='', key='fqdn')
```

Comment:
I think that it's better to store required fields in the class metadata. For example we can introduce a `_required_fields` static field in each WAPI object class. For example:
```python
class DNSZone(InfobloxObject):
    _required_fields = ["fqdn"]
```

Reply:
There is a lot of WAPI objects, that have required fields.

This way it will be easier for us to introduce new required fields, and to apply such bug fixes to all WAPI objects.